### DISCUSSION TITLE
Dialogs

### DISCUSSION THREAD

**Discussion by sbc640964** [inertiajs/inertia#1244] [Ideas]
In the past they talked about a dialog feature. ([tree](https://github.com/inertiajs/inertia/tree/dialogs))

Are you working on it?
Don't know why, but it seems the last change was a long time ago (14 months)

**Comment by godismyjudge95** [comment]
Yes, they are still working on it.  If you want early access to it you can sponsor:  https://github.com/sponsors/reinink
This is the same process they did with SSR.  They work on it and provide it exclusively to sponsors for testing until it is ready for 1.0 release.

**Comment by sbc640964** [comment]
I see @reinink closed the pull request https://github.com/inertiajs/inertia/pull/642.
 Does that mean it's ready?

**Reply by reinink** [reply]
Nope, sorry! We're doing a big cleanup of old issues and PRs right now in effort to start with a bit of a "fresh slate", so a bunch of things got closed, including the original dialogs PR, which unfortunately does not mean it's done. I still hope to revisit the dialogs feature some day, but right now my focus is squashing bugs in the library.

**Reply by ngl5000** [reply]
Didn't this get demo'd years ago as though it was a finished feature? I was excited to try it and wasted a lot of time searching the documentation.
